Implement developers mode

TiBiBa opened the pull request:
Provide a general summary of your changes in the Title of the PR

**Description**

In this PR we implement a developers mode to give users the option to hide all "learning" elements that they don't need when focussing just on coding. By pressing the `toggle()` the commands and adventure tabs will be hidden and the code and output window will be increased in width and height.

**Fix for**

This PR fixed #1276.

**How to test**

Press the toggle() button below the "run code" button and notice that all non-coding elements of the page will disappear. Press it again to return to the original view. Notice that the toggle will not be remembered when switching levels.

**Screenshots**
![Schermafbeelding 2021-11-15 om 14 05 24](https://user-images.githubusercontent.com/38155424/141786900-5b356330-53d4-4422-8de4-3595ee780ad0.png)
![Schermafbeelding 2021-11-15 om 14 05 32](https://user-images.githubusercontent.com/38155424/141786908-bf5b5826-cbfc-4b8b-bcfa-a40eba1ece9d.png)

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
- [ ] Links to an existing issue or discussion (if not, create an issue first)
- [ ] Describes changes clear in the format above (present tense, no subject)
- [ ] Has a "how to test" section
- [ ] Only one thing is done in this pull request (specifically please try to refrain from mixing textual changes to the yamls from code changes)

Felienne commented on templates/incl-editor-and-output.html:
```diff
@@ -106,3 +106,16 @@
+      <div class="dot absolute w-6 h-6 bg-white rounded-full shadow -left-1 -top-1 transition" style="top: -5px;"></div>
+    </div>
+    <div class="ml-3">
+      Developers mode
```
Shall we add this to texts and localize it? I guess for Dutch kids this is ok but for Greek or Chinese kids we want the option to show it in their own lang.

TiBiBa replied:
Will add a localization based on the *.yaml files and will let you know when it's finished.

TiBiBa replied:
Fixed!

Felienne reviewed (changes requested):
Super awesome @TiBiBa and so quickly made!

Just want to have the name of the mode localized and we are good to go.

Felienne commented:
Awesome @TiBiBa, just changed the wording for kids a bit because they might not know the word developer/ontwikkelaar to mean programmer.

 I just have one tiny request, when in Turtle mode, there now are quite some lines for text, can we limit that to 2 or 3 lines, so there is more room for drawing:

![image](https://user-images.githubusercontent.com/1003685/141834684-7a2e2655-661e-4ba5-afca-d94477d35e78.png)

TiBiBa commented:
> Awesome @TiBiBa, just changed the wording for kids a bit because they might not know the word developer/ontwikkelaar to mean programmer.
> 
> I just have one tiny request, when in Turtle mode, there now are quite some lines for text, can we limit that to 2 or 3 lines, so there is more room for drawing:
> 
> ![image](https://user-images.githubusercontent.com/1003685/141834684-7a2e2655-661e-4ba5-afca-d94477d35e78.png)

Fixed this with the last commit!

Felienne commented:
Kijk eens @OnnoEbbens, u vraagt wij draaien hoor :D Gaat zo live op de site.
